lib/wovnrb.rb

```diff
@@ -85,7 +85,12 @@ def switch_lang(body, values, url, lang=@store.settings['default_lang'], headers
       lang = Lang.new(lang)
       ignore_all = false
       new_body = []
-      body.each do |b|
+
+      # generate full_body to intercept
+      full_body = ''
+      body.each { |b| full_body += b }
+
+      [full_body].each do |b|
```

Comment:
Can we remove this iteration since it is no longer needed?

Reply:
I thought about it too but I liked to keep with iterations as body is always supposed to be an iterator

Reply:
ok